ydb/docs/en/core/dev/troubleshooting/performance-issues.md

```diff
@@ -0,0 +1,12 @@
+# Performance issues
+
+Database performance issues can be classified into several categories based on their nature:
+
+- **Reaching resource limits**. These issues refer to situations when the physical resources allocated to a database system — such as CPU, memory, disk space, and network bandwidth — are insufficient to handle the demands of the workload.
+
+- **Schema design issues**. These issues refer to inefficient decisions that were made during the creation of database schemas. Such issues can significantly impact database performance.
+
+- **Query performance issues**. These issues refer to problems that arise when database queries execute slower that expected because of their inefficient design.
+
+
```

Comment:
Another potential major category is YDB operating in a degraded mode, like during a major outage or drills.